Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -160,6 +173,207 @@ YUI.add('ez-subitemlistview', function (Y) {
+         *
+         * @method _hideEditIcon
+         * @protected
+         * @param {Object} e event facade
+         */
+        _hideEditIcon: function (e) {
+            var selectedCell = this._getPriorityCell(e.target.getAttribute('data-location-id'));
+
+            selectedCell.removeClass('ez-subitem-hovered-priority-cell');
+        },
+
+        /**
+         * Hide the error icon.
+         *
+         * @method _hideErrorIcon
+         * @protected
+         * @param {Object} input the DOM node of the input
+         * @param {Object} selectedCell the DOM node of the current selected cell
+         */
+        _hideErrorIcon: function (input, selectedCell) {
+            selectedCell.removeClass('ez-subitem-error-priority-cell');
+            selectedCell.addClass('ez-subitem-selected-priority-cell');
+        },
+
+        /**
+         * Show the 'validate' & 'cancel' buttons which allow to update or not the priority.
+         *
+         * @method _displayPriorityButtons
+         * @protected
+         * @param {Object} e event facade
+         */
+        _displayPriorityButtons: function (e) {
+            var selectedCell = this._getPriorityCell(e.target.getAttribute('data-location-id'));
+
+            e.preventDefault();
+            if (!this.get('isOnePrioritySelected')) {
+                this._set('isOnePrioritySelected', true);
+                selectedCell.removeClass('ez-subitem-hovered-priority-cell');
+                selectedCell.addClass('ez-subitem-selected-priority-cell');
+                e.target.removeAttribute('readonly');
+            }
+        },
+
+        /**
+         * Hide the 'validate' & 'cancel' buttons which allow to update or not the priority.
+         * Also fill back priority input with the current sub item priority.
+         *
+         * @method _restorePriorityCell
+         * @protected
+         * @param {Object} e event facade
+         */
+        _restorePriorityCell: function (e) {
+            e.preventDefault();
+            this._hidePriorityButtons();
+            this._restorePriorityValue(e);
+        },
+
+        /**
+         * Restore the input value with the priority of the sub item
+         *
+         * @method _restorePriorityValue
+         * @protected
+         * @param {Object} e event facade
+         */
+        _restorePriorityValue: function (e) {
+            var input = this._getPriorityInput('#' + e.target.getAttribute('data-priority-input')),
+                subItem = this._getSubItem(input.getAttribute('data-location-id'));
+
+            input.set('value', subItem.location.get('priority'));
+            input.setAttribute('readonly', 'readonly');
+        },
+
+        /**
+         * Return the sub item with the giiven locationId
+         *
+         * @method _getSubItem
+         * @protected
+         * @param {Number} locationId
+         */
+        _getSubItem: function (locationId) {
```

Comment:
I would add `Struct` to the name of the method in a the description to reflect that it's returns what is called `struct` in the `subitems` attribute description.
Also the document for the returned value is missing.